schemas/code_signature.yml

```diff
@@ -57,3 +57,24 @@
         This is useful for logging cryptographic errors with the certificate validity or trust status.
         Leave unpopulated if the validity or trust of the certificate was unchecked.
       example: ERROR_UNTRUSTED_ROOT
+      
+    - name: team_id
+      level: extended
+      type: keyword
+      short: The team identifier used to sign the process.
+      description: >
```

Comment:
I think for both of these, it's worth noting that it's mostly relevant to macOS or other Apple operating systems. It might not be intuitive for people trying to map this field to a Windows sensor, like Elastic Endpoint or Sysmon.

Reply:
Makes sense.

I had initially wondered if we could make these fields more generic for use across different OS types, but based on what @Trinity2019 [shared](https://github.com/elastic/ecs/issues/1249#issuecomment-772062111), Windows and macOS don't seem to align enough.

Reply:
I added suggested comment: https://github.com/elastic/ecs/pull/1269/commits/5a6d1b20e1865d265956859a996b433294154183
Will fix other `make` generated files once you guys approve this change^

Reply:
LGTM

Reply:
Thanks @ebeahan !